src/SFA.DAS.EmployerFinance/Data/Scripts/Script0008CreateAccountPayeSchemes.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE [dbo].[AccountPayeSchemes]
+(
+  [Id] BIGINT NOT NULL IDENTITY,
+  [AccountId] BIGINT NOT NULL,
+  [EmployerReferenceNumber] VARCHAR(16) NOT NULL,
+  [Created] DATETIME2 NOT NULL,
+  [Deleted] DATETIME2 NULL,
+  CONSTRAINT [PK_AccountPayeSchemes] PRIMARY KEY CLUSTERED ([Id] ASC),
+  CONSTRAINT [FK_AccountPayeSchemes_Accounts_AccountId] FOREIGN KEY ([AccountId]) REFERENCES [Accounts] ([Id]),
+  CONSTRAINT [UK_AccountPayeSchemes_AccountId_EmployerReferenceNumber] UNIQUE ([AccountId] ASC, [EmployerReferenceNumber] ASC),
```

Comment:
I think a unique index might be needed instead along the lines of `WHERE [Deleted] IS NULL`.

Reply:
Indeed! Fixed.